operator/helper/parser.go

```diff
@@ -154,6 +163,10 @@ func (p *ParserOperator) ParseWith(ctx context.Context, entry *entry.Entry, pars
 		severityParseErr = p.SeverityParser.Parse(entry)
 	}
 
+	if p.TraceParser != nil {
+		_ = p.TraceParser.Parse(entry)
```

Comment:
We need to handle the error like `time` and `severity`.

Reply:
Admittedly, the error handling below probably needs to be reworked a bit, but I think it's ok for this PR to just add another individual check:

```
if timeParseErr != nil {
    return p.HandleEntryError(ctx, entry, errors.Wrap(timeParseErr, "time parser"))
}
if severityParseErr != nil {
    return p.HandleEntryError(ctx, entry, errors.Wrap(severityParseErr, "severity parser"))
}
if traceParseErr != nil {
    return p.HandleEntryError(ctx, entry, errors.Wrap(traceParseErr, "trace parser"))
}
```